examples/vision/melanoma_classification_with_tfrecords.py

```diff
@@ -0,0 +1,438 @@
+"""
+Title: Melanoma Classification with TFRecords
+Author: Amy MiHyun Jang
+Date created: 2020/07/29
+Last modified: 2020/07/29
+Description: Classifying skin lesion images from TFRecords for melanoma diagnosis
+"""
+"""
+# 1. Introduction + Set Up
+
+Transfer learning in computer vision is the process of using a pre-trained model on a new
+dataset of images. This is extremely useful when the dataset is not that big. ML models
+thrive when there is a lot of data available to train on, but it many scenarios,
+obtaining large datasets is quite difficult. The pre-trained model is trained on a big
+dataset, and the weights are preserved. The trained layers of the model is used as the
+basis for our new model. We will be using the VGG16 model to predict whether or not a
```

Comment:
VGG16 is fairly outdated (2014). Prefer using EfficientNet, ResNet50, or Xception for transfer learning